### PR TITLE
Updated hercule and deal with mutliple files

### PR DESCRIPTION
**Pull request by jmatsushita**
This plugin is now able to deal with multiple files!

### CODE REVIEW

File: tasks/hercule.js [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
-			}
-		);
+		var num_files = this.files.length
+		this.files.forEach(function(f){
+			hercule.transcludeFile(f.src[0],
+				{ relativePath: path.dirname(path.join(process.cwd(), f.src[0])) },
+				function callback(err,output) {
+
+					if (err) {
+						console.log(err)
```
Comment (chesleybrown) [inline tasks/hercule.js]: Recommend using grunt's logging instead of `console.log`.

File: tasks/hercule.js [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
+						console.log(path.join(process.cwd(), f.src[0]))
+						console.log(path.dirname(path.join(process.cwd(), f.src[0])));
+					} else {
+						grunt.file.write(path.join(process.cwd(), f.dest), output);
+					}
+
+					num_files = num_files - 1
+
+					// grunt.log.write(f.src[0]);
```
Comment (chesleybrown) [inline tasks/hercule.js]: Just remove these lines if they are no longer needed/used.

**Comment by chesleybrown** [comment]
The travis build is also failing 😢

**Comment by jmatsushita** [comment]
It's fixed ! :)

**Comment by chesleybrown** [comment]
Cool thanks! 👍

**Comment by chesleybrown** [comment]
Published as `grunt-hercule@2.0.0`.

**Pull request merged**